### DISCUSSION TITLE
How to install: little video tutorial on youtube would be great

### DISCUSSION THREAD

**Discussion by bzaman** [imartinez/privateGPT#502] [Ideas]
if anyone interested. Thank you

**Comment by joshuachris2001** [comment]
with a little bit of looking, there is a video about it. [[https://www.youtube.com/watch?v=A3F5riM5BNE](https://www.youtube.com/watch?v=A3F5riM5BNE)]
but I don't see why a video is needed when the readme breaks down the python install quite simply.

**Reply by bzaman** [reply]
Thanks, simple enough! 🙏

**Comment by AlonSpitz** [comment]
Really good one: https://www.youtube.com/watch?v=jxSPx1bfl2M&ab_channel=MatthewBerman

**Reply by bzaman** [reply]
Bravo Zulu @AlonSpitz. Very details. 💪
